.github/workflows/check-shell.yml

```
@@ -6,19 +6,19 @@ jobs:
     strategy:
       matrix:
         os: [
-          ubuntu-latest,
-          # intel mac
-          macos-13,
-          # apple silicon
-          macos-latest
-        ]
+            ubuntu-latest,
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 10 but found 12

(indentation)